packages/run-it/src/components/ConfigForm/ConfigForm.tsx

```diff
@@ -259,6 +259,7 @@ export const ConfigForm: FC<ConfigFormProps> = ({
     await adaptor.login()
   }
 
+  /* eslint-disable */
```

Comment:
Can make this specifically `no-nested-ternary`, so that other errors can be caught, or may, since nested ternaries seem to be popular around here, just allow it via a global override?

Reply:
> since nested ternaries seem to be popular around here, just allow it via a global override?

I like that. I will spin up another PR later on today.